src/sequelize.ts

```diff
@@ -51,12 +52,16 @@ export function BigNumberStringType (propName: string): Partial<ModelAttributeCo
   }
 }
 
-export default function (app: Application): void {
+export default async function (app: Application): Promise<void> {
```

Comment:
Interesting that linter doesn't yell about not using `await` 🤔

Reply:
Had `await` here: https://github.com/rsksmart/rif-marketplace-cache/pull/269/files/a34fa72a7815c31bd81e6a900e46e2f4dcf090f6#diff-f39990cee8f2282dd02df78076ac6a13R61

Reply:
Oh missed that one 😅